h/static/styles/help-page.scss

```diff
@@ -38,16 +38,29 @@ a {
 
 .columns {
   display: flex;
+  justify-content: space-between;
   flex-direction: row;
 }
 
 .columns__col-1-2 {
   flex-shrink: 1;
-  width: 50%;
+  width: 45%;
+}
 
-  margin-left: 20px;
-  &:first-child {
-    margin-left: 0;
+.wrap-columns {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  margin-bottom: 2.5em;
+}
```

Comment:
Note for myself: The purpose of this is to display the images next to each other if there is space or above otherwise.